docs/metrics.asciidoc

```diff
@@ -160,3 +160,56 @@ All metrics collected from `prometheus_client` are prefixed with `"prometheus.me
 [[prometheus-metricset-beta]]
 ===== Beta limitations
  * The metrics format may change without backwards compatibility in future releases.
+
+[float]
+[[custom-metrics]]
+=== Custom Metrics
+
+Custom metrics allow you to send your own metrics to Elasticsearch.
+
+The most commond way to send custom metrics is with the Prometheus metrics set.
+See <<prometheus-metricset>> for more information.
```

Comment:
I would just turn the `Prometheus metrics set` text in the previous sentence into a link and omit this sentence.